src/test/java/com/artipie/npm/proxy/NpmProxyTest.java

```diff
@@ -200,8 +201,13 @@ void setUp(final Vertx vertx, final VertxTestContext context)
         throws IOException, InterruptedException {
         this.server = prepareServer(vertx, context);
         this.storage = new InMemoryStorage();
+        final YamlMapping yaml = Yaml.createYamlMappingBuilder()
+            .add("remote-url", String.format("http://localhost:%d", NpmProxyTest.DEF_PORT))
```

Comment:
@brastak I think we need to use `reserver-network-port` Maven goal to find available and accessible by user port to avoid random test failures: https://www.mojohaus.org/build-helper-maven-plugin/reserve-network-port-mojo.html

Reply:
@g4s8 Do we actually need this plugin? We use this approach in the VertxSliceServerTest:
try (ServerSocket socket = new ServerSocket(0)) {
    return socket.getLocalPort();
}
Plugin do the same.
Anyway, there is no any guarantees that port will be really available (we get free port, release it, try to open it one more time. It can be taken somebody else in the middle between 2 and 3)

Reply:
@brastak ok, let's use new socket now, we may change it later if needed